roswallpexpo.txt

```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = ReactOS Wallpaper Exposition
+Version = 1
+License = CC BY
+Description = A wallpaper set (apple, leaves, sky etc.), by fastcom.
+Category = 16
+URLSite = https://jira.reactos.org/browse/CORE-12639
+URLDownload = https://jira.reactos.org/secure/attachment/67588/ReactOSWallpaperExposition-v1.zip
```

Comment:
Does this work in ReactOS ?

Reply:
![image](https://github.com/reactos/rapps-db/assets/2107452/d97b16a2-ddb1-4da4-aff5-30133f459074)
Oh, unsupported compression method. I will retry.

Reply:
And we didn't support JPEG wallpapers.

Reply:
![VirtualBox_ReactOS_17_03_2024_13_45_26](https://github.com/reactos/rapps-db/assets/2107452/00007f99-8ba8-4e7c-989e-a78244527e55)
Now it works!